testing/src/main/java/io/airlift/testing/Closeables.java

```diff
@@ -11,6 +11,11 @@ private Closeables()
     {
     }
 
+    /**
+     * @deprecated Usage of this method is discouraged, as it may hide real problems.
+     * Use {@link #closeAll(Closeable...)} instead and, if necessary, suppress exceptions
+     * explicitly.
+     */
     public static void closeQuietly(Closeable... closeables)
```

Comment:
Shouldn't this also have an `@deprecated` annotation also?

Reply:
Right.